src/Libraries/RevitServices/Persistence/ElementBinder.cs

```diff
@@ -152,16 +152,25 @@ public override int GetHashCode()
             //concat the strings and int ids into one string and get hashcode and xor
             //a multiserializableID with same IDs in different order will return not equal  
             return (StringIDs == null ? 0 : StringIDs.Aggregate((i, j) => i + " " + j).GetHashCode()) ^
-             (IntIDs == null ? 0: IntIDs.Select(x => x.ToString()).Aggregate((i, j) => i + " " + j).GetHashCode());
+             (IntIDs == null ? 0 : IntIDs.Select(x => x.ToString()).Aggregate((i, j) => i + " " + j).GetHashCode());
 
-         }
         }
 
+        /// <summary>
+        /// this method tests if this multiSerializableId is contained in another
+        /// </summary>
+        /// <param name="other"></param>
+        /// <returns></returns>
+        public virtual bool isSubset(MultipleSerializableId other)
```

Comment:
This method should check that the entire set of ids is a subset of the `other`, by checking if the set difference is empty this only returns true in the case that the entire set of Ids is a subset in the other, not only if there is an intersection.